src/UnderlineNav/UnderlineNavLink.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import Box from '../Box'
+import {merge} from '../sx'
+import {get} from '../constants'
+
+export const UnderlineNavLink = ({
+  sx: sxProp = {},
+  href = '#',
+  children,
+  selected = false,
+  leadingIcon: LeadingIcon,
+  ...props
```

Comment:
Should we support the `as` prop to allow people to use React Router-like link components?